### DISCUSSION TITLE
Post-quantum certs cannot used in Tquic

### DISCUSSION THREAD

**Discussion by Ericyu0412** [Tencent/tquic#411] [General]
我尝试使用liboqs生成了后量子算法签名的证书，它们的大小大概在5KB-20KB之间，然后使用tquic-example-c项目中的代码，将其中的证书替换成生成的证书，编译后./simple-server运行后直接退出，这是因为使用的UDP的payload太小导致的吗，其次，我尝试自定义QUIC的token值，并构建一个短签名的认证机制，该从哪个函数中尝试修改并生成？

**Comment by iyangsj** [comment]
目标暂不支持后量子算法签名的证书

**Reply by Ericyu0412** [reply]
那应该如何自定义token字段呢
